docs/topics/i18n/translation.txt

```diff
@@ -851,6 +851,12 @@ You can then access the information::
     Name in English: {{ lang.name }}<br />
     Bi-directional: {{ lang.bidi }}
 
+.. versionadded:: 1.9
```

Comment:
It would be better to incorporate the new features in the existing examples and add an annotation after that describes what changed.  For example, this one might say:
``
.. versionadded:: 1.9
   The ``name_translated`` attribute was added.
```